content/docs/04-clusters/01-public-cloud/01-aws/02-architecture.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Architecture"
+metaTitle: "AWS Architecture with Palette"
+metaDescription: "Learn about Palette and the architecture used to support Palette"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# AWS IaaS Architecture
+
+
+The following are some architectural highlights of the Amazon Web Services (AWS) clusters, provisioned by Palette:
+
+
+- Kubernetes nodes can be distributed across multiple availability zones (AZs) to achieve high availability (HA). For each of the AZs that you select, a public subnet and a private subnet is created.
+
+
+- All the control plane nodes and worker nodes are created within the private subnets, so there is no direct public access available.
+
+
+- A Network Address Translation (NAT) Gateway is created in the public subnet of each AZ, to allow nodes in the private subnet to be able to go out to the internet or call other AWS services.
+
+
+- An Internet Gateway (IG) is created for each Virtual Private Cloud (VPC), to allow Secure Shell Protocol (SSH) access to the bastion node for debugging purposes. SSH into Kubernetes nodes is only available through the bastion node. A bastion node helps to provide access to the Amazon Elastic Compute Cloud (EC2) instances. This is because the EC2 instances are created in a private subnet and the bastion node operates as a secure, single point of entry into the infrastructure. The bastion node can be accessed via SSH or Remote Desktop (RDP).
```

Comment:
`An Internet Gateway (IG) is created for each Virtual Private Cloud (VPC) to allow Secure Shell Protocol (SSH) access to the bastion node for debugging purposes. SSH into Kubernetes nodes is only available through the bastion node. In addition, a bastion node helps to provide access to the Amazon Elastic Compute Cloud (EC2) instances. This is because the EC2 instances are created in a private subnet, and the bastion node operates as a secure, single point of entry into the infrastructure. The bastion node can be accessed via SSH or Remote Desktop (RDP).
		`